app/experimenter/experiments/constants.py

```diff
@@ -114,6 +116,7 @@ def FEATURE_TYPE_CHOICES(cls):  # pragma: no cover
         (STATUS_ACCEPTED, STATUS_ACCEPTED_LABEL),
         (STATUS_LIVE, STATUS_LIVE_LABEL),
         (STATUS_COMPLETE, STATUS_COMPLETE_LABEL),
+        (STATUS_REJECTED, STATUS_REJECTED),
```

Comment:
Same here, should this be `(STATUS_REJECTED, STATUS_REJECTED_LABEL),`? (assuming we rename above reference on L110)